src/devices/xiaomi.ts

```diff
@@ -977,7 +977,7 @@ const definitions: Definition[] = [
         whiteLabel: [{vendor: 'Xiaomi', model: 'RLS-K01D'}],
         description: 'Aqara Zigbee 3.0 LED strip T1',
         extend: [
-            light({effect: false, powerOnBehaviour: false, colorTemp: {startup: false, range: [153, 370]}}),
+            light({effect: false, powerOnBehaviour: false, colorTemp: {startup: false, range: [153, 370]}, color: {modes: ['xy']}}),
```

Comment:
```suggestion
            light({effect: false, powerOnBehaviour: false, colorTemp: {startup: false, range: [153, 370]}, color: true}),
```

Does this also work?

Reply:
Should [do the same](https://github.com/Koenkk/zigbee-herdsman-converters/blob/79795781289bd5b2a576b4e18cdfc1abfdf58264/src/lib/modernExtend.ts#L230), I just instinctively used the [old value](https://github.com/Koenkk/zigbee-herdsman-converters/blob/79795781289bd5b2a576b4e18cdfc1abfdf58264/src/lib/exposes.ts#L688).

PS: I don't have that device, I just found the bug.